mission-vision-values.md

```diff
@@ -0,0 +1,108 @@
+# OTel Mission, Vision, and Values
+
+## Mission (our overall north star as a community)
+
+OpenTelemetry's Mission: **to enable effective observability through
+ubiquitous, portable, high-quality telemetry.**
+
+## Vision (the world we imagine for OTel end-users)
+
+Excellent observability requires high-quality telemetry, as well as the
+performant, consistent instrumentation that makes it possible.
+
+Telemetry in this context is the firehose of raw observational data streaming
+out of software applications, and while “high-quality telemetry” may be a
+requirement for excellent observability, it’s still unreasonable and
+unrealistic to expect developers of application software to add or maintain the
+necessary instrumentation on their own. That is a massive undertaking, and in
+practice it’s not “just code” – there is also necessary alignment around
+protocols and semantic conventions for tags, attributes, and other metadata to
+consider.
+
+So how do we get high-quality, turnkey telemetry without a massive,
+unsustainable engineering effort? This is where OpenTelemetry comes in. To
+achieve our vision, OpenTelemetry sees five key opportunities, listed here:
+
+### Telemetry should be easy
+
+With OpenTelemetry we want high-quality telemetry to be easy, especially for
+end-users. That means that OpenTelemetry should have fast time-to-value, set
+reasonable defaults yet allow for customization, and pair with excellent
+documentation and a top-tier overall developer experience.
+
+### Telemetry should be universal
+
+Telemetry protocols and conventions should be unified across languages and
+signal types (tracing, metrics, logging, etc), not divergent or siloed. This
+means that OpenTelemetry aspires to find technical solutions that work
+consistently, both locally and globally.
+
+### Telemetry should be vendor-neutral
+
+For decades, monitoring and observability vendors have created differentiation
+– and lock-in – through agents that automatically gather useful telemetry from
+a wide integration surface. This lock-in is limiting for customers and inhibits
+innovation by tightly coupling telemetry collection and telemetry storage and
+analysis. With OpenTelemetry, we strive to provide a level playing field for
+all observability providers, avoiding lock-in for any vendor and interoperating
+with other OSS projects in the telemetry and observability ecosystem.
+Furthermore, with the looming threat of software-supply-chain attacks, we aim
```

Comment:
This last line referencing software-supply-chain-attacks feels like a periphery line item to a larger subject of vendor-neutral.  I've not seen a statement about software security. Honestly, I may have missed it as I read through this doc.

Reply:
yeah, fair point – this sentence was added in response to an earlier reviewer comment. I'll try to allude to the broader benefits of OSS without getting so far into the weeds on this.